README.md

```diff
@@ -50,6 +50,12 @@ No permissions required (`saveToPhotos` requires permission [check](#note-on-fil
 
 Note: This library does not require `Manifest.permission.CAMERA`, if your app declares as using this permission in manifest then you have to obtain the permission before using `launchCamera`.
 
+#### Targeting below Android API v30
+
+Check if your application meets the requirement for AndroidX Photo Picker, otherwise add the entry to your `AndroidManifest.xml`: <https://developer.android.com/training/data-storage/shared/photopicker#device-availability>
```

Comment:
To be honest this backwards compatibility part for me is not exactly clear from the docs on how it should work. On the compatible Android APIs it works as expected, and the Photo Picker is showing up.

> Did the photo picker actually work for you in the example app on Android <30?

For my Android 8 Samsung device, the fallback documents picker showed up, instead of the new Photo Picker :/ 

I tried with your suggested changes, but on an Android API 30 emulator, still the system documents picker was showing up.

---
I honestly have no idea how this backwards compatibility should be configured to work correctly, the Android docs seem too vague.

Reply:
I managed to get the backported version to work on older Android versions 🎊 

What (primarily) did the trick for me was to test on physical devices (via BrowserStack App Live). I don't know where the limitation in the emulators is. Maybe it has to do with Google Play Services not being updated/installed 🤷 

Once, I got this on one of my physical devices (Samsung Galaxy S20 Ultra, Android 10) and I believe this is where it downloads/installs the backported version of the photo picker:
<img src="https://github.com/user-attachments/assets/c3fb459b-a03a-453b-b74c-67c94b86f474" width="200" alt="screenshot-Samsung Galaxy S20 Ultra-10 0">

Here is proof that the new photo picker works on a Samsung Galaxy S10, Android 9 device:
<img src="https://github.com/user-attachments/assets/a9c1081f-ae68-4e54-892c-87340ddb21f9" width="200" alt="screenshot-Samsung Galaxy S10-9 0 (1)">

Reply:
In addition to testing on a physical device, I also had to add this to `example/android/app/build.gradle` (due to reasons stated earlier in this thread):
```
...
implementation("com.facebook.react:flipper-integration")

// androidx.activity:activity >= 1.7.0 is required for the backported photo picker to work on Android < 30 https://developer.android.com/jetpack/androidx/releases/activity#1.7.0. 
// This dependency can be removed from here once the react-native library updates androidx.appcompat:appcompat to >= 1.7.0
// https://github.com/facebook/react-native/blob/3dfe22bd27429a43b4648c597b71f7965f31ca65/packages/react-native/gradle/libs.versions.toml#L11
// because androidx.appcompat:appcompat:1.7.0 depends on androidx.activity:activity:1.7.0 https://developer.android.com/jetpack/androidx/releases/appcompat#1.7.0
implementation("androidx.activity:activity:1.7.+")

if (hermesEnabled.toBoolean()) {
...
```

As this only fixes the example app, I think we should also add a comment about this to the README under the new `Targeting below Android API v30` section.

People can test what version of `androidx.activity:activity` their app depends on with `./gradlew app:dependencies` and then look for the first occurrence of `androidx.activity:activity`. Without `implementation("androidx.activity:activity:1.7.+")` it looks like this for the example app:
```
+--- com.facebook.react:react-android -> 0.73.6
|    +--- androidx.appcompat:appcompat:1.6.1
|    |    +--- androidx.activity:activity:1.6.0
```

With `implementation("androidx.activity:activity:1.7.+")` it looks like this:
```
+--- com.facebook.react:react-android -> 0.73.6
|    +--- androidx.appcompat:appcompat:1.6.1
|    |    +--- androidx.activity:activity:1.6.0 -> 1.7.2
```
In the README, I don't know if we should go this much into details about how to find out about your current version of `androidx.activity:activity` or if it is enough to say something like: 
> If your `minSdkVersion` is < 30 and your app doesn't already depend on/include `androidx.activity:activity:1.7.+` then you need to add `implementation("androidx.activity:activity:1.7.+")` in the `dependencies` block of your `app/build.gradle` file in order to support the backported photo picker on Android < 30.

Reply:
I can confirm that on both my Android API <30 shows the backported Photo Picker with the `implementation("androidx.activity:activity:1.7.+")` line added 🎊 

<details>
<summary>Samsung Galaxy A7 2018 - API 29 "Q"</summary>

> Manually dragged the Picker a bit down so the app is visible. Otherwise it fills the whole screen.

![Samsung SM-A750F](https://github.com/user-attachments/assets/7a4bc265-dbec-457f-bdc7-cd5ad52b40cd)

</details>

<details>
<summary>Samsung Galaxy A5 2017 - API 26 "Oreo"</summary>

> Manually dragged the Picker a bit down so the app is visible. Otherwise it fills the whole screen.

![Samsung SM-A520F](https://github.com/user-attachments/assets/73d38e43-6489-4871-aeb9-999b0c790997)

</details>

If I do not add the line, the fallback documents selector shows up.

---

I have updated the readme to show the tricks needed. Also added your suggested comment in the example app's `build.gradle`.